bundles/net.tourbook.cloud/src/net/tourbook/cloud/dropbox/DropboxFolderBrowser.java

```diff
@@ -81,6 +81,7 @@
     */
```

Comment:
I could not create this comment in line 62 therefore the comment is here:

I have prefixed all my "Dialog" classes with `Dialog...` in the classname

Reply:
Fixed